### PR TITLE
Add the option to skip the build mojo from the command line

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
This is rather trivial. @aloubyansky @gsmet could you please review?

File: devtools/maven/src/main/java/io/quarkus/maven/BuildMojo.java [hunk 1]
```diff
@@ -116,7 +116,8 @@
     @Parameter(property = "ignoredEntries")
     private String[] ignoredEntries;
 
-    @Parameter(defaultValue = "false")
+    /** Skip the execution of this mojo */
+    @Parameter(defaultValue = "false", property = "quarkus.skip")
```
Comment (gastaldi) [inline devtools/maven/src/main/java/io/quarkus/maven/BuildMojo.java]: Perhaps we should name it `quarkus.build.skip`? Other mojos might follow the same convention for the skip flag too :wink:

Comment (ppalaga) [reply devtools/maven/src/main/java/io/quarkus/maven/BuildMojo.java]: +1

**Comment by ppalaga** [comment]
0e04db1 now uses `quarkus.build.skip` property name instead of the originally proposed  `quarkus.skip`.

**Review by gastaldi — APPROVED** [review]

**Comment by gastaldi** [comment]
@ppalaga thanks! Do you need that for 1.3.0.Final or can it wait for 1.4 ?

**Comment by ppalaga** [comment]
> @ppalaga thanks! Do you need that for 1.3.0.Final

1.3.0.Final would be highly welcome, thanks!

**Pull request merged**